types/react-native/index.d.ts

```diff
@@ -1225,6 +1225,11 @@ export interface TextInputAndroidProps {
      * Vertically align text when `multiline` is set to true
      */
     textAlignVertical?: 'auto' | 'top' | 'bottom' | 'center';
+    
+    /**
+     * Prevent the soft keyboard from showing when the field is focused if false. The default value is true.
```

Comment:
Shall we use the documentation as on the React Native website you linked? It sounds a bit easier to understand :)

Reply:
sure, changing that now